src/SecurityHubOverviewLambda/SecurityHubOverview-function.ts

```diff
@@ -0,0 +1,26 @@
+// ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
+import * as path from 'path';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { Construct } from 'constructs';
+
+/**
+ * Props for SecurityHubOverviewFunction
+ */
+export interface SecurityHubOverviewFunctionProps extends lambda.FunctionOptions {
+}
+
+/**
+ * An AWS Lambda function which executes src/SecurityHubOverviewLambda/SecurityHubOverview.
```

Comment:
Ook deze comment, als je enigszins bekend bent met onderstaande is het wel helder dat ie een lambda op dat path toevoegt. Liever: Wat doet die lambda, waarom hebben we 'm?

Reply:
Ook deze is gegenereerd door projen :)